pyschema/source_generation.py

```diff
@@ -78,48 +183,50 @@ def _class_source(schema, indent):
 
 class CachedGraphTraverser(object):
     def __init__(self):
-        self.descendents = {}
+        self.descendants = {}
         self.started = set()
 
-    def find_descendents(self, a):
-        if a in self.descendents:
+    def find_descendants(self, a, max_depth=sys.getrecursionlimit()):
```

Comment:
I used sys.getrecusionslimit() as a standin for math.inf, feels ugly but makes the code easy to work with